test/bugs-reported/bug1.chpl

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
On second thought I think this may be more appropriate as a JIRA issue, as there will probably be some discussion. Here's a link to our JIRA: https://chapel.atlassian.net/projects/CHAPEL/issues/